packages/components/src/components/hds/alert/index.ts

```diff
@@ -178,8 +155,16 @@ export default class HdsAlert extends Component<HdsAlertSignature> {
     const actions = element.querySelectorAll(
       `${CONTENT_ELEMENT_SELECTOR} button, ${CONTENT_ELEMENT_SELECTOR} a`
     );
-    if (actions.length) {
+
+    const isRealAlert: boolean =
```

Comment:
[thought] what does "real" mean in this context? is there a more semantic way to call this variable?

Reply:
It means an Alert which actually alerts users (has role="alert" & aria-live="polite").

I didn't actually know what to call it and realize the name may seem a bit "weird" perhaps so I'm open to better naming ideas.

Reply:
@alex-ju @zamoore Do you have ideas for clearer naming for this variable (if you don't think it's clear)?

Reply:
`isSemanticAlert` maybe? Everything else I immediately thought of was funny but probably not useful.

Reply:
@MelSumner Well both you and Cristiano mentioned the word "semantic" so maybe I'll just change it to that. Thanks!

Reply:
Updated it. Thanks for the feedback.